.gitpod.yml

```diff
@@ -0,0 +1,31 @@
+tasks:
+  - name: Stargate
+    command: docker-compose up stargate
+  - name: api
+    command: |
+      sleep 35
+      docker-compose up eddiehub-api
+  - name: Eddiebot
+    init: |
+      npm i -g npm
```

Comment:
Maybe we can run this in a custom dockerfile to not have to run it every time on init